richardltyler/ideabox-boilerplate
# Feature/html and css

PR 
## Pull Request

<br>

### Description

##### What does this PR do?

* Builds HTML skeleton and CSS styling

##### What does it fix?

* N/A

##### Is this a feature or a fix?

* feature

##### Where should the reviewer start?

* HTML and CSS

##### How should this be tested?

* in terminal, open index.html
* check format, spacing, font, colors against comp
 
<br>

***
